spinta/backends/__init__.py

```diff
@@ -1,11 +1,12 @@
 import contextlib
 
-from spinta.types import Type
+from spinta.types import Type as MetaType
+from spinta.types.type import Type
 from spinta.components import Context, Model
 from spinta.commands import error, prepare
 
 
-class Backend(Type):
+class Backend(MetaType):
```

Comment:
`spinta.types.Type` is deprecated, please remove it and leave just `Backend:`.

Reply:
then remove `error(Exception, Context, Backend, MetaType, **kwargs)` command too?

Reply:
Read my next comment about `error()`.